code/modules/clothing/suits/armor.dm

```diff
@@ -224,7 +224,7 @@
 	icon_state = "custodian"
 	item_flags = THICKMATERIAL | COVER_PREVENT_MANIPULATION
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
-	armor = list(melee = 30, bullet = 30, energy = 25, bomb = 25, bio = 100, rad = 80)
+	armor = list(melee = 35, bullet = 30, energy = 25, bomb = 25, bio = 100, rad = 80)
```

Comment:
No

Reply:
Well I thought that since it's basically "janitor armor but church made" and normal janitors get to fight roaches (since afair they are supposed to "clean pests") I thought having at least baseline melee protection was good for them
Can you at least tell me why "not" on this particular armor above all else?